plugins/modules/flightctl_resource_info.py

```
@@ -225,7 +224,7 @@ def main():
     try:
         result = module.get_one_or_many(options)
     except FlightctlException as e:
-        module.fail_json(msg=f"Failed to get resource: {e}")
+        module.fail_json(msg=f"Failed to get resource: {e}", error=traceback.format_exc())
```

Ruff (0.8.2)

Undefined name `traceback`

(F821)